api/util/ipnet/ipnet.go

```diff
@@ -0,0 +1,120 @@
+// Package ipnet wraps net.IPNet to get CIDR serialization.
+package ipnet
+
+import (
+	"encoding/json"
+	"net"
+	"strings"
+
+	"github.com/pkg/errors"
```

Comment:
we don't really use this package even though is vendored, can we please drop the import here and keep api/ package to have a less deps as possible?

Reply:
oh sure, didn't even notice that. Blast from the past.